galactic/distribution.yaml

```
@@ -535,6 +535,21 @@ repositories:
       url: https://github.com/eclipse-cyclonedds/cyclonedds.git
       version: iceoryx
     status: maintained
+  data_buffer:
+    doc:
+      type: git
+      url: https://github.com/OUXT-Polaris/data_buffer.git
+      version: master
```

Should this be master` or `ros2` to match below. 
```suggestion
      version: ros2
```